packages/select/src/common/predicate.ts

```diff
@@ -14,10 +14,20 @@
 export type ItemListPredicate<T> = (query: string, items: T[]) => T[];
 
 /**
- * Customize querying of individual items. Return `true` to keep the item, `false` to hide.
- * This method will be invoked once for each item, so it should be performant. For more complex
- * queries, use `itemListPredicate` to operate once on the entire array.
+ * Customize querying of individual items.
  *
- * If defined with `itemListPredicate`, this prop will be ignored.
+ * __Filtering a list of items.__ This function is invoked to filter the list of
```

Comment:
it's weird to have the exact same documentation block in two places. it's especially weird to reference the prop names here, when we're not in a react component class module... can you simplify this to a simple one-sentence description and tell people to "see its usage sites in `IListItemsProps` ..." ?